lib/protocols.js

```diff
@@ -1001,6 +1025,21 @@
     }
   }());
 
+  var WrappedMessage = (function() {
+    var root=protobuf.loadSync(path.join(__dirname+'/protostream/message-wrapping.proto'));
```

Comment:
Will the `message-wrapping.proto` file be shipped with the library? I mean are we sure that the `loadSync` will find it the lib will be installed on the user side?

Reply:
I think it will because the path of the file is not relative. Also I have tested it.